perception/autoware_detected_object_validation/schema/obstacle_pointcloud_based_validator.schema.json

```
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Obstacle Pointcloud Based Validator",
+  "type": "object",
+  "definitions": {
+    "obstacle_pointcloud_based_validator": {
+      "type": "object",
+      "properties": {
+        "min_points_num": {
+          "type": "array",
+          "items": {
+            "type": "integer"
+          },
+          "default": [10, 10, 10, 10, 10, 10, 10, 10],
+          "description": "The minimum number of obstacle point clouds in DetectedObjects"
+        },
+        "max_points_num": {
+          "type": "array",
+          "items": {
+            "type": "integer"
+          },
+          "default": [10, 10, 10, 10, 10, 10, 10, 10],
+          "description": "The max number of obstacle point clouds in DetectedObjects"
+        },
+        "min_points_and_distance_ratio": {
+          "type": "array",
+          "items": {
+            "type": "integer"
+          },
+          "default": [800, 800, 800, 800, 800, 800, 800, 800],
```

```suggestion
            "type": "number"
          },
          "default": [800.0, 800.0, 800.0, 800.0, 800.0, 800.0, 800.0, 800.0],
```
In source code, `min_points_and_distance_ratio` declares as `double`, so we need to follow that type.